app/build.gradle.kts

```diff
@@ -55,8 +55,8 @@ android {
 
     lint {
         lintConfig = rootProject.file(".lint/config.xml")
-        isCheckAllWarnings = true
-        isWarningsAsErrors = true
+        // isCheckAllWarnings = true
```

Comment:
Do we want to leave this commented it out or take them out at all? Is there a plan to reactivate them?

Reply:
Good catch, these are actually deprecated and produce compilation errors but I will search for the alternative.

Reply:
I see you updated those in buildSrc/src/main/kotlin/commons/android-library.gradle.kts but this are still commented, we wanna delete them or update this as well?

Reply:
Oh right, I did not see that there are two configurations. It's solved now.